Rlog/View/WorkSpace/WorkSpaceListView.swift

```diff
@@ -7,14 +7,92 @@
 
 import SwiftUI
 
+
 struct WorkSpaceListView: View {
+    
     var body: some View {
-        Text(/*@START_MENU_TOKEN@*/"Hello, World!"/*@END_MENU_TOKEN@*/)
+        NavigationView {
+            ZStack {
+                Color.cardBackground
+                    .ignoresSafeArea()
+                
+                ScrollView {
+                    ForEach(models, id: \.self) { model in
+                        WorkSpaceCell(model: model)
+                    }
+                }
+                .padding(.top, 32)
+                .navigationBarTitleDisplayMode(.inline)
+                .toolbar {
+                    ToolbarItem(placement: .navigationBarLeading) {
+                        Text("근무지")
+                            .font(.title2)
+                            .fontWeight(.bold)
+                    }
+                    ToolbarItem(placement: .navigationBarTrailing) {
+                        Button(action: { print("button pressed") }){
+                            Image(systemName: "plus")
+                                .fontWeight(.bold)
+                        }
+                    }
+                }
+            }
+        }
     }
 }
 
-struct WorkSpaceListView_Previews: PreviewProvider {
-    static var previews: some View {
-        WorkSpaceListView()
+struct CustomModel: Hashable {
+    let name: String
+    let hourlyWage: Int
+    let date: String
+    let hasJuhyu: Bool
+    let hasTax: Bool
+    let Color: String
+    let workDays: String
+    let schedules: String
+    
+    func text(info: WorkSpaceInfo) -> String {
```

Comment:
```suggestion
    // TODO: CoreData model에도 추가해야함.
    func getValue(from info: WorkSpaceInfo) -> String {
```

이 method는 CoreData Model에도 추가가 필요한 method 일까요 ? 맞다면, TODO의 추가가 필요해보입니다.
+ method/parameter의 이름을 조금 변경하면 어떨지도 함께 제안드립니다 !

Reply:
제안해주신 함수명 변경 적용했습니다.
CoreData Model에 추가 해야할 method인지는 @insub4067 과 의논 후 별도의 PR로 적용하겠습니다.
일단 우선 주석으로 TODO 작성했습니다.
 [88202cb](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Bremen-DD/pull/8/commits/88202cb326bcef04c4f97808bebcf51c3e869402)